src/components/SmartAppBanner/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect } from 'react'
+import { useConfig } from '../../contexts/ConfigContext'
+import PropTypes from 'prop-types'
+
+export const SmartAppBanner = (props) => {
+  const {
+    UIComponent,
+    storeAndroidId,
+    storeAppleId,
+    storeKindleId
+  } = props
+
+  const [{ configs }] = useConfig()
+
+  useEffect(() => {
+    if (!configs) return
+    if (storeAndroidId || configs?.android_app_id?.value) {
+      const metaGoogle = document.createElement('meta')
```

Comment:
@YisusJe  you can optimize this methods, you're repeating the same code for all options. You can create an array and do the same thing once